Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Dockerfile to run e2e integration tests against a test PocketBase server
 FROM node:16-alpine3.16
 
-ARG POCKETBASE_VERSION=0.15.0
+ARG POCKETBASE_VERSION=0.23.0-rc12
```

Comment:
Another RC was just released, may as well update this 
```suggestion
ARG POCKETBASE_VERSION=0.23.0-rc13
```

Reply:
Done